docs/pages/experiments/docs/codeblock.md

```diff
@@ -42,3 +42,25 @@ npm install @mui/data-grid-premium
 ## Component
 
 {{"component": "modules/components/HighlightedCodeWithTabs", "tabs": [{"tab":"JS", "code":"<div>Hello</div>", "language": "jsx"}, {"tab": "TS", "code": "type A = {}"}]}}
+
+## With header path
+
+<div class="codeblock-with-header">
+  <p class="header">PlainCssSliderDeep1.js</p>
+
+```jsx
```

Comment:
Would it be easier to do this?
```suggestion
```jsx header="PlainCssSliderDeep1.js"
```

This is how tailwind is doing https://github.com/tailwindlabs/tailwindcss.com/blob/master/src/pages/docs/responsive-design.mdx#L131.

Reply:
I think we should follow [Docusaurus](https://docusaurus.io/docs/markdown-features/code-blocks#code-title). It's simpler.